microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -35,17 +35,27 @@ The Payment Request API⁠⁠ handles the actual payment transaction when a purc
 See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
 
 
+<!-- ====================================================================== -->
+## Origin trial and flag
+
+There's an origin trial and Edge flag for this feature.  As of December 1, 2024, this origin trial is likely to end soon, graduating the feature to be incorporated in Microsoft Edge.  See [Experimental features and origin trials for PWAs](https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials).
+
+The Edge Experiments flag in `edge://flags/`<!-- todo: which flag? --> will be enabled by default, when this feature ships in Edge Stable.
+
+This feature ships with a pre-release of Microsoft Edge.<!-- todo: specify the exact version number here when available -->
```

Comment:
specify the exact Edge version number here when this feature is available
@runyuany

Reply:
comment probably Outdated, will Close & re-enter as needed